28-05-2015/alphabet/repeat_char_format.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './alphabet'
+
+class RepeatCharFormat
+  
+  def initialize(character, number)
+  	@characters = character.letters(number)
+  end
+
+  def format
+    @characters.zip(@characters).shuffle.flatten.join('')
```

Comment:
`shuffle` is not formatting. It is an ordering. How would you implement a requirement where the alphabets are ordered evenly, leaving every other character like `ACEG...`?